crates/sui-types/src/messages.rs

```diff
@@ -327,6 +444,15 @@ pub struct TransactionData {
     pub gas_budget: u64,
 }
 
+impl DeterministicSizeForGasMetering for TransactionData {
+    fn deterministic_size_for_gas_metering(&self) -> u128 {
+        let consts_sum = size_of::<SuiAddress>() as u128
+            + size_of::<ObjectRef>() as u128
+            + 2 * (size_of::<u64>() as u128);
```

Comment:
Could you include a comment for what these numbers are?

Reply:
Will do.